src/adminactions/api.py

```diff
@@ -61,19 +61,20 @@ def merge(master, other, fields=None, commit=False, m2m=None, related=None):  #
     """
         Merge 'other' into master.
 
-        `fields` is a list of fieldnames that must be readed from ``other`` to put into master.
-        If ``fields`` is None ``master`` will get all the ``other`` values except primary_key.
+        `fields` is a list of fieldnames that must be read from ``other`` to put into master.
         Finally ``other`` will be deleted and master will be preserved
 
     @param master:  Model instance
     @param other: Model instance
-    @param fields: list of fieldnames to  merge
+    @param commit: bool
```

Comment:
Feels like the order of parameter types should match the order in the parameter list of the function?